laterpay/signing.py

```diff
@@ -1,15 +1,16 @@
 # -*- coding: utf-8 -*-
-from __future__ import absolute_import, print_function
-
 import hashlib
 import hmac
+import warnings
 
 import six
 from six.moves.urllib.parse import quote, urlparse
+from furl.omdict1D import omdict
```

Comment:
So, you're importing this here but this dependency is declared on `requirements-test.txt` instead of `setup.py`.